app/views/entities/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= stylesheet_link_tag 'entityIndex' %>
```

Comment:
- [OPTIONAL] Kindly follow the design guidelines for the header on the entities page, Add a new group, and Add a new entity:
   - [ ] The header color is different.
   - [ ] The header text is white.